From ec9d0e471f85582ef600c86f2e7f3cb3094cbed7 Mon Sep 17 00:00:00 2001
From: Tianhao Chai <cth451@gmail.com>
Date: Wed, 30 Nov 2022 16:03:46 -0500
Subject: [PATCH 2/3] dpdk 22.11: RTE_DEV_TO_PCI is no longer possible.

There are no public API from DPDK that allows extraction of PCIe vendor
ID / device ID anymore.
---
 lib/netdev-dpdk.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/lib/netdev-dpdk.c b/lib/netdev-dpdk.c
index fe2e58c79..647bc2bd7 100644
--- a/lib/netdev-dpdk.c
+++ b/lib/netdev-dpdk.c
@@ -3663,13 +3663,6 @@ netdev_dpdk_get_status(const struct netdev *netdev, struct smap *args)
     uint16_t vendor_id = RTE_PCI_ANY_ID;
     uint16_t device_id = RTE_PCI_ANY_ID;
     bus = rte_bus_find_by_device(dev_info.device);
-    if (bus && !strcmp(rte_bus_name(bus), "pci")) {
-        pci_dev = RTE_DEV_TO_PCI(dev_info.device);
-        if (pci_dev) {
-            vendor_id = pci_dev->id.vendor_id;
-            device_id = pci_dev->id.device_id;
-        }
-    }
     ovs_mutex_unlock(&dpdk_mutex);
 
     smap_add_format(args, "port_no", DPDK_PORT_ID_FMT, dev->port_id);
-- 
2.38.1

